src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,71 @@
-import React from 'react';
+import classNames from 'classnames';
+import { useState } from 'react';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 import './FindMovie.scss';
+import { MovieData } from '../../types/MovieData';
+
+type Props = {
+  addMovie: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState('');
+  const [queryError, setQueryError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    try {
+      e.preventDefault();
+      setIsLoading(true);
+
+      const result = await getMovie(query);
+
+      if (Object.prototype.hasOwnProperty.call(result, 'Error')) {
+        setQueryError(true);
+      } else {
+        const data = result as MovieData;
+
+        setMovie({
+          title: data.Title,
+          description: data.Plot,
+          imgUrl: data.Poster === 'N/A'
+            ? 'https://via.placeholder.com/360x270.png?text=no%20preview'
+            : data.Poster,
+          imdbUrl: `https://www.imdb.com/title/${data.imdbID}`,
+          imdbId: data.imdbID,
+        });
+      }
+    } catch (error: unknown) {
+      // eslint-disable-next-line no-alert
+      alert(`Network Error: ${error}`);
```

Comment:
if I use console.error, eslint throws an error. So that the linter and type script do not give errors, it was possible to implement it only in this way - throw new Error('Network error');

Reply:
Yea, it's better)